File: backend/backend-analyzer/backend-analyzer/Models/ReviewStatistic.cs
using System;

namespace AnalyzerServer.Models
{
    public class ReviewStatistic
    {
        public int Id { get; set; }

        public decimal AverageRating { get; set; } = 0.00m;

        public string MostPopularKeyword { get; set; }

        public DateTime LastUpdated { get; set; } = DateTime.UtcNow;
    }
}

File: backend/models/reviewStatistic.js
const { Sequelize, DataTypes } = require('sequelize');

module.exports = (sequelize, DataTypes) => {
    return sequelize.define('ReviewStatistic', {
        average_rating: { type: DataTypes.DECIMAL(3, 2), defaultValue: 0.00 },
        most_popular_keyword: { type: DataTypes.STRING(50) },
        last_updated: { type: DataTypes.DATE, defaultValue: Sequelize.NOW },
    }, { tableName: 'review_statistics', timestamps: false });
};
